drivers/python/rethinkdb/net.py

```diff
@@ -515,6 +523,12 @@ def __init__(self, conn_type, host, port, db, auth_key, timeout, ssl, **kwargs):
         self._instance = None
         self._next_token = 0
 
+    def client_port(self):
+        return self._instance._socket.client_port()
```

Comment:
> I think it would be slightly better to have this be a property.

I was thinking this too, actually. @deontologician should we make these read-only properties in the JS driver as well?